client/src/components/ProTx/components/maps/MainMap.js

```diff
@@ -146,12 +131,54 @@ function MainMap({
         },
         maxNativeZoom: 14 // All tiles generated up to 14 zoom level
       });
+
+      // Handle
+
       if (dataLayer && layersControl) {
         // we will remove data layer from mapand from control
         layersControl.removeLayer(dataLayer);
         dataLayer.remove();
       }
 
+      newDataLayer.on('click', e => {
+        const clickedGeographicFeature =
+          e.layer.properties[GEOID_KEY[geography]];
+
+        if (refSelectedGeoid.current) {
+          newDataLayer.resetFeatureStyle(refSelectedGeoid.current);
+        }
+
+        if (clickedGeographicFeature !== refSelectedGeoid.current) {
+          updateSelectedGeographicFeature(clickedGeographicFeature);
+          const highlightedStyle = {
+            ...getFeatureStyle(
+              mapType,
+              data,
+              metaData,
+              geography,
+              year,
+              clickedGeographicFeature,
+              observedFeature,
+              maltreatmentTypes
+            ),
+            color: 'red',
```

Comment:
I would suggest this should be 'black' for now because a) the red color has a psychological value connotation we do not want to imply and b) the black looks cleaner and is easier to see against the transparent coloring of the region.

Reply:
<img width="667" alt="Screen Shot 2021-06-28 at 1 40 58 PM" src="https://user-images.githubusercontent.com/3238078/123687419-878d5a00-d816-11eb-85fa-31799297234b.png">

Reply:
Compare these:
<img width="629" alt="Screen Shot 2021-06-28 at 1 41 48 PM" src="https://user-images.githubusercontent.com/3238078/123687572-b3a8db00-d816-11eb-8499-be60efc95411.png">
<img width="603" alt="Screen Shot 2021-06-28 at 1 42 14 PM" src="https://user-images.githubusercontent.com/3238078/123687590-b73c6200-d816-11eb-92b3-ece1bb48781a.png">